pingcap/tiflow
# sink(ticdc): fix daemon dbus process not stopped issue

### What problem does this PR solve?


Issue Number: close #4926

### What is changed and how it works?
 check DBUS_SESSION_BUS_ADDRESS first and then try to discovery it.
 if DBUS_SESSION_BUS_ADDRESS is found, do nothing, go-dbus will not create daemon-dbus,
if not, then create daemon-dbus, and set DBUS_SESSION_BUS_ADDRESS env and record the pid
kill the created process when cdc command is stopped

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test 
 add DBUS_SESSION_BUS_ADDRESS env, check if the daemon process is stopped
 remove DBUS_SESSION_BUS_ADDRESS env, check if the daemon process is stopped





### Release note 

```release-note
`None`.
```
